### PR TITLE
Removes the trace packet

### PR DESCRIPTION
**Pull request by DeJayDev**
**Please describe the changes this PR makes and why it should be merged:**
This PR removes the trace packet as per https://github.com/discordapp/discord-api-docs/pull/967.

> Later, at some point in the future, the `_trace` field will be dropped.
\- zorkian

**Status**
- [x] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [ ] This PR changes the library's interface (methods or parameters added)
- [x] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/client/websocket/WebSocketShard.js [hunk 1]
```diff
@@ -368,9 +361,8 @@ class WebSocketShard extends EventEmitter {
         this.emit(ShardEvents.READY);
 
         this.sessionID = packet.d.session_id;
-        this.trace = packet.d._trace;
         this.status = Status.READY;
-        this.debug(`READY ${this.trace.join(' -> ')} | Session ${this.sessionID}.`);
+        this.debug(`READY Session ${this.sessionID}.`);
```
Comment (amishshah) [inline src/client/websocket/WebSocketShard.js]:
```suggestion
        this.debug(`READY | Session ${this.sessionID}.`);
```

File: src/client/websocket/WebSocketShard.js [hunk 2]
```diff
@@ -381,10 +373,9 @@ class WebSocketShard extends EventEmitter {
          */
         this.emit(ShardEvents.RESUMED);
 
-        this.trace = packet.d._trace;
         this.status = Status.READY;
         const replayed = packet.s - this.closeSequence;
-        this.debug(`RESUMED ${this.trace.join(' -> ')} | Session ${this.sessionID} | Replayed ${replayed} events.`);
+        this.debug(`RESUMED Session ${this.sessionID} | Replayed ${replayed} events.`);
```
Comment (amishshah) [inline src/client/websocket/WebSocketShard.js]:
```suggestion
        this.debug(`RESUMED | Session ${this.sessionID} | Replayed ${replayed} events.`);
```

Comment (amishshah) [reply src/client/websocket/WebSocketShard.js]: Sorry for the nitpicking!

Comment (DeJayDev) [reply src/client/websocket/WebSocketShard.js]: Nitpicking is appreciated, I had no clue how I wanted these to look.

**Comment by DeJayDev** [comment]
GitHub picks really gross names for those inline resolves. My bad.

**Pull request merged**